capsules/extra/src/nonvolatile_storage_driver.rs

```diff
@@ -128,10 +280,90 @@ impl Default for App {
             command: NonvolatileCommand::UserspaceRead,
             offset: 0,
             length: 0,
+            has_requested_region: false,
+            region: None,
         }
     }
 }
 
+// the following helper functions are used for converting to/from
+// u8 slices that are read/written to nonvolatile storage
+fn u8_slice_to_usize(bytes: &[u8]) -> usize {
+    let mut result: usize = 0;
+
+    for (i, &byte) in bytes.iter().enumerate() {
+        result |= (byte as usize) << (8 * i);
+    }
+
+    result
+}
```

Comment:
I believe you can use `usize::from_le_bytes` and similar for the methods here

Reply:
Changed in https://github.com/tock/tock/pull/4109/commits/d02e6325cc22b2eeb101dac1dbdfd6263c2636a2.